index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <!-- end of google fonts -->
+    <!-- fontawesome -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <!-- end of fontawesome -->
+    <!-- style css -->
+    <link rel="stylesheet" href="style.css" />
+    <!-- end of style css -->
+    <title>Emmanuel</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="logo">
+        <a href="index.html"><h1 class="logo">LOGO</h1></a>
+      </div>
+
+      <div class="menu-bar">
+        <i class="fa-solid fa-bars"></i>
+      </div>
```

Comment:
- [ ] Please kindly wrap this hamburger element with `nav` tag as suggested earlier. The `nav` tag is for semantics reason and great user experience for mobile devices. 👍 
```suggestion
      <nav class="menu-bar">
        <i class="fa-solid fa-bars"></i>
      </nav>
```